File: week-6-python/lecture-6/agree.py
s = input("Do you agree? ")

# Using or
# if s == "Y" or s == "y":
#     print("Agreed.")
# elif s == "N" or s == "n":
#     print("Not agreed.")

# Using list
s = s.lower()
if s in ["y", "yes"]:
    print("Agreed.")
elif s in ["n", "no"]:
    print("Not agreed.")

File: week-1-c/lecture/agree.c
#include <stdio.h>

int main(void)
{
  char c;

  printf("Do you agree? ");
  scanf("%c", &c);

  if (c == 'Y' || c == 'y')
  {
    printf("yes, I am agree\n");
  }
  else if (c == 'N' || c == 'n')
  {
    printf("no, I am disagree\n");
  }

  return 0;
}